File: root/data/src/main/java/org/sc/data/model/StaticTrailDetails.java
package org.sc.data.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class StaticTrailDetails {
    public static final String PATH_GPX = "pathGpx";
    public static final String PATH_KML = "pathKml";
    public static final String PATH_PDF = "pathPdf";

    private String pathGpx;
    private String pathKml;
    private String pathPdf;
}

File: root/data/src/main/java/org/sc/data/model/StaticTrailDetails.kt
package org.sc.data.model

data class StaticTrailDetails(
    var pathGpx : String,
    var pathKml : String,
    var pathPdf : String
) {
    companion object {
        const val PATH_GPX = "pathGpx"
        const val PATH_KML = "pathKml"
        const val PATH_PDF = "pathPdf"
    }
}